test/units/module_utils/common/test_removed.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019, Andrew Klychkov @Andersson007 <aaklychkov@mail.ru>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible.module_utils.common.removed import removed_module
+
+
+@pytest.mark.parametrize('input_data', [u'2.8', 2.8, 2, '', ])
+def test_removed_module_sys_exit(input_data):
+    """Test for removed_module function, sys.exit()."""
+
+    with pytest.raises(SystemExit) as wrapped_e:
+        removed_module(input_data)
+
+    assert wrapped_e.type == SystemExit
+    assert wrapped_e.value.code == 1
+
+
+@pytest.mark.parametrize(
+    'input_data, expected_msg, expected_warn',
+    [
+        (
+            u'2.8',
+            u'This module has been removed.  '
```

Comment:
This can be changed to look for only one space now that #57209 is merged. Thanks again for writing tests. Much appreciated.

Reply:
done, thank you